app/views/games/show.html.erb

```diff
@@ -0,0 +1,26 @@
+<body>
+ <div class="container">
+ <table class="block-center">
+     <% 4.times do |row| %>
+      <tr class="even">
+       <% 8.times do |even_cell| %>
+       <% if @pieces.where(file: even_cell + 1, rank: row * 2 + 1)[0]%>
+        <td><font class="unicode"><%= @pieces.where(file: even_cell + 1, rank: row * 2 + 1)[0].image.html_safe %></font></td> 
```

Comment:
Stylistic, but IMO it looks much better if each HTML element creates a visual nesting w/ indentation similar to Ruby def/end blocks. Makes it easier to spot what's going on.
```
<td>
    <font class="unicode">
        <%= ...
```